usaspending_api/common/etl/spark.py

```diff
@@ -261,3 +261,47 @@ def merge_delta_table(spark: SparkSession, source_df: DataFrame, delta_table_nam
         WHEN NOT MATCHED THEN INSERT *
     """
     )
+
+
+def diff(left: DataFrame, right: DataFrame, unique_key_col="id", compare_cols=[], collect=False):
+    """
+    Compares two dataframes that share a schema and returns difference at a row level. 
+    Meant to be run in a databricks notebook.
```

Comment:
Please remove this line in the comment. While it _can_ be run there, it doesn't have to be, and would actually require a fair bit of knowledge to get our code installed in a platform like that to get it to be imported and run (i.e. it's more likely someone would just copy-paste it where they want it). Also we don't want to provide code that is platforms specific if we can avoid it or suggest that the code is platform specific (this code currently is not).

Reply:
I'll remove the comment.